utilities/pipelines/resourceRemoval/helper/Invoke-ResourceRemoval.ps1

```diff
@@ -150,7 +150,31 @@ function Invoke-ResourceRemoval {
             }
             break
         }
+        'Microsoft.Resources/tags' {
+            # Get current tags on the subscription
+            $subscriptionId = $resourceId.Split('/')[2]
+            $currentTags = $(Get-AzTag -ResourceId /subscriptions/$subscriptionId).Properties
+
+            # Get the tags to remove
+            $tagsToRemove = @('Test', 'TestToo')
```

Comment:
Tags provided by deployment validation tests are just for testing purposes and shouldn't be hardcoded in the CI environment utilities. 

If the entire tags object gets deleted even if `onlyUpdate` is set to `true`, then I agree that an issue should be opened to address it. @ntaheij could you please open a bug in the board explaining the actual and indended behavior?

Reply:
Thanks for the response. Currently, yes, `onlyUpdate` does delete all tags. Issue is that, when deploying the tests for the module, currently all tags are being deleted, which means tags for i.e. billing are not available. I will open a bug.

Reply:
If there is any way to only delete the tags that are created, I'd be interested to use that.

Reply:
Thanks @ntaheij! I'm going to put the PR in draft for the moment and reference the PR in the issue

Reply:
Note, I replied in the issue :)